images/access-setup/content/bin/setup_kcp.sh

```diff
@@ -138,7 +138,9 @@ generate_kcp_credentials() {
   kubeconfig="$credentials_dir/kcp/$kcp_name.kubeconfig"
 
   printf "    - Create workspace:\n"
-  kubectl kcp workspace use "$KCP_ORG"
+  if [[ "$(kubectl kcp workspace current | cut -d\" -f2)" != "$KCP_ORG" ]]; then
```

Comment:
I don't see why it is useful, tested with the latest 0.6 release (server and client):
~~~
$ kubectl kcp workspace use root:pipelines-service
Current workspace is "root:pipelines-service".
$ kubectl kcp workspace use root:pipelines-service
Current workspace is "root:pipelines-service".
$ kubectl kcp workspace use root
Current workspace is "root".
$ kubectl kcp workspace use root:pipelines-service
Current workspace is "root:pipelines-service".
~~~

Reply:
Strange, I have a different behavior:
```
> ./images/access-setup/content/bin/setup_kcp.sh -w ~/Code/appStudio/infra/kcp-stable --kubeconfig ~/Code/appStudio/infra/infra/work/credentials/kubeconfig/kcp/kcp.kubeconfig  -d
...
+ kubectl kcp workspace use root:rh-sso-15826143
error: workspaces.tenancy.kcp.dev "rh-sso-15826143" not found
```

I used kcp release-0.6 (commit 789d0ca1f14eebc783558ce7178044e80e890158), and made sure to recompile.

Bharath is creating an image, let's wait to see if I can reproduce using the image.

Reply:
The image is missing a few things, which I've corrected in this PR.

I've reverted this specific change, even though I still see the issue using the new image. If the issue is real, I can take care of it in a separate PR if necessary.